enzyme/Enzyme/Utils.cpp

```diff
@@ -158,6 +158,98 @@ Function *getOrInsertDifferentialFloatMemcpy(Module &M, PointerType *T,
   return F;
 }
 
+Function *getOrInsertMemcpyStrided(Module &M, PointerType *T, unsigned dstalign,
+                                   unsigned srcalign, Value *stride) {
+  Type *elementType = T->getElementType();
+  assert(elementType->isFloatingPointTy());
+  std::string name = "__enzyme_memcpy_" + tofltstr(elementType) + "da" +
+                     std::to_string(dstalign) + "sa" +
+                     std::to_string(srcalign) + "stride" +
+                     std::to_string(cast<ConstantInt>(stride)->getSExtValue());
```

Comment:
Don't do a cast to constantint -- it may not be constant

Reply:
Could you please help me figure out an alternate way to make the name of the function unique for that particular stride number?